src/type/fraction/function/fraction.js

```diff
@@ -37,7 +37,7 @@ export const createFraction = /* #__PURE__ */ factory(name, dependencies, ({ typ
    *
    *    bignumber, number, string, unit
    *
-   * @param {number | string | Fraction | BigNumber | Array | Matrix} [args]
+   * @param {number | string | Fraction | BigNumber | Unit | Array | Matrix} [args]
```

Comment:
the ```fraction``` type in the index.d.ts file needs to be update as well :))

Reply:
Good point, I've updated the TypeScript definitions